spec.md

```diff
@@ -1598,6 +1605,54 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 | Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListSnapshots` operation again with an empty `starting_token`. |
 
 
+#### `ControllerExpandVolume`
+
+A Controller plugin MUST implement this RPC call if plugin has `EXPAND_VOLUME` capability. 
+This RPC allows the CO to expand size of underlying volume.
+
+This call can be made by the CO during any time in the lifecycle of the volume after creation.
+If plugin has `EXPAND_VOLUME` node capability, then `NodeExpandVolume` MUST be called after `ControllerExpandVolume` has completed.
+If volume can not be expanded when in-use - plugin should return `FAILED_PRECONDITION` error and CO should retry only after calling `ControllerUnpublishVolume` if plugin has `PUBLISH_UNPUBLISH_VOLUME` controller capability.
```

Comment:
.. and what if PUBLISH_UNPUBLISH_VOLUME isn't a supported capability, what does recovery look like then?

Reply:
Fixed